src/main/java/callofcactus/menu/GameScreen.java

```diff
@@ -59,7 +61,7 @@
 	private BackgroundRenderer backgroundRenderer;
 	private SpriteBatch AIBatch;
 	//  MAP variables
-	private Map map;
+	private CallOfCactusMap defaultMap;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Perhaps 'defaultMap' could be replaced by a local variable.](https://www.codacy.com/app/guushamm/Call-of-Cactus/file/1385244494/issues/source?bid=2979575&fileBranchId=2980149#l64)